src/pb_stub.cc

```diff
@@ -711,15 +743,19 @@ main(int argc, char** argv)
   py::scoped_interpreter guard{};
 
   stub->Initialize(model_version, argv[6] /* triton install path */);
-  bool non_graceful_exit = false;
+  std::atomic<bool> non_graceful_exit(false);
 
-  bool background_thread_running = true;
+  std::atomic<bool> background_thread_running = {true};
```

Comment:
More of a cleanliness fix but I think you should stick to the same convention for initializing these atomic bool variables ie. () or {} but not both.